llvm/test/Transforms/InstCombine/shift-logic.ll

```diff
@@ -433,9 +433,9 @@ define i8 @shl_sub(i8 %x, i8 %y) {
 define i8 @shl_sub_multiuse(i8 %x) {
 ; CHECK-LABEL: @shl_sub_multiuse(
 ; CHECK-NEXT:    [[SH0:%.*]] = shl i8 [[X:%.*]], 3
-; CHECK-NEXT:    [[R:%.*]] = sub i8 [[SH0]], 42 ; constant operand on the 'sub'
```

Comment:
Uh, did update_llc_test_checks.py really add that comment?...

Reply:
It removed the comment, not added it @nikic

Reply:
It added the comment as it was in the original comment, but when it was transformed, llc removed it.

Reply:
I know, but why was it there in the first place. The previous commit added it, but that suggests the first commit is wrong.

And please do not resolve conversations off the bat like that.

Reply:
Because I put it there originally.

Reply:
Well don't then?

And for the second time, stop resolving conversations like that.